astroquery/mast/core.py

```diff
@@ -430,20 +288,20 @@ def _request(self, method, url, params=None, data=None, headers=None,
             The response from the server.
         """
 
-        startTime = time.time()
-        allResponses = []
-        totalPages = 1
-        curPage = 0
+        start_time = time.time()
+        all_responses = []
+        total_pages = 1
+        cur_page = 0
 
-        while curPage < totalPages:
+        while cur_page < total_pages:
```

Comment:
wow, I haven't appreciated this before. Would be a nice feature request for upstream to ask to an API feature that returns everything without paging.

Reply:
Our new-style APIs don't require this, but the old style will persist.... for while. 😛

Reply:
oh, I wish all my wishes ever were working out this smoothly 😉